server/app/lib/actions/fusor/deployment/rhev/ose_launch.rb

```diff
@@ -30,20 +30,36 @@ def plan(deployment)
           def run
             ::Fusor.log.info '====== OSE Launch run method ======'
             deployment = ::Fusor::Deployment.find(input[:deployment_id])
+            os_name_rhel71 = 'RedHat 7.1'
+            os_name_rhel72 = 'RedHat 7.2'
+            ptable_name = 'Openshift Ptable'
+            hostgroup = 'OpenShift'
```

Comment:
Regarding tests, created a trello card to track it https://trello.com/c/xS5LLRS3